### DISCUSSION TITLE
OR - Tool with OSRM Facing accuracy issue

### DISCUSSION THREAD

**Discussion by Shivara2022** [google/or-tools#3726] [Routing (and legacy CP) questions] [answered]
I'm using OSRM as map dataset to solve CVRP using OR tool. When I plotted a store co-ordinates in both OSRM & Google maps.

There was an huge difference in store placement in map with some 600 meters or more difference. In this case solving with these kind of data will not give an effective path right.

Any suggestions?

**Comment by jmarca** [comment, marked as answer]
Edit open street map to correct the mistakes. Then reload your osrm instance with the fixed OSM data. JamesOn Mar 29, 2023, at 19:17, Shivara2022 ***@***.***> wrote:﻿
I'm using OSRM as map dataset to solve CVRP using OR tool. When I plotted a store co-ordinates in both OSRM & Google maps.
There was an huge difference in store placement in map with some 600 meters or more difference. In this case solving with these kind of data will not give an effective path right.
Any suggestions?

**Reply by Shivara2022** [reply]
Thank you so much for your help. I'm new to this.

Correction you mean by using any login & directly edit the main data and export the dataset? Can you share any supporting links to follow the steps. Also I'm having some 3000+ customer data thinking the possibility of correcting all.

Thanks

**Comment by jmarca** [comment]
Open street map is its own project. Go there for help https://wiki.openstreetmap.org/wiki/Main_Page
JamesOn Mar 29, 2023, at 19:37, Shivara2022 ***@***.***> wrote:﻿
Thank you so much for your help. I'm new to this.
Correction you mean by using any login & directly edit the main data and export the dataset? Can you share any supporting links to follow the steps. Also I'm having some 3000+ customer data thinking the possibility of correcting all.
Thanks
